ld-pruning-wf.wdl

```diff
@@ -0,0 +1,440 @@
+version 1.0
+
+# General notes:
+#
+# variables and functions exclusive to the WDL version tend to have "py_" in front
+# and use camelCase between words - this is to make it clear they don't quite 
+# correlate to the CWL
+
+# [1] ld_pruning -- calculates linkage diseq on a GDS file
+task ld_pruning {
+	input {
+		File gds
+		File? sample_include_file
+		File? variant_include_file
+		String genome_build = "hg38"  # can also be hg18 or hg19
+		Float ld_r_threshold = 0.32
+		Float ld_win_size = 10  # yes, a float, not an int
+		Float maf_threshold = 0.01
+		Float missing_threshold = 0.01
```

Comment:
That would break the python section, as inline python in a WDL cannot detect if something is defined. It'd end up just being a lot of checks of empty strings. I think this method is more explicit and clear for users.

Reply:
Also, if I'm not mistaken, the CWL has sbdefaultvalue which matches these values.